drodrpg/DRODLib/DbHolds.cpp

```diff
@@ -3139,8 +3159,15 @@ MESSAGE_ID CDbHold::SetProperty(
 			const HoldStatus eMatchHoldsWithStatus =
 					(info.typeBeingImported == CImportInfo::Player ||
 							info.typeBeingImported == CImportInfo::SavedGame)
-					? Main : NoStatus;	//match player imports for demo and full main hold
-			const UINT dwLocalHoldID = GetLocalID(eMatchHoldsWithStatus);
+					? Tendry : NoStatus;	//match player imports for demo and full main hold
+			//Across DLC data, there could be multiple instances of the same author GID.
+			//To join a DLC hold, we need to match against all of these.
+			std::pair<PrimaryKeyMultiMap::const_iterator, PrimaryKeyMultiMap::const_iterator> localIDMap =
+				info.PlayerIDMap.equal_range(this->dwPlayerID);
+			CIDSet localPlayerIDs;
+			for (PrimaryKeyMultiMap::const_iterator it = localIDMap.first; it != localIDMap.second; ++it)
+				localPlayerIDs += it->second;
+			const UINT dwLocalHoldID = GetLocalID(eMatchHoldsWithStatus, localPlayerIDs, this->dwPlayerID);
```

Comment:
Putting a comment here because it's a bit I'm a little unsure about. I did the changes using TSS as a guide, and this is the one place it looks for a specific hold status rather than checking if it's an official hold.

Also, the DLC data checks were added.

Reply:
Ah -- multiple author profiles that are actually the same author (my intuition says this can occur across, say, multiple DLC .dat files that replicate the same player profile as a hold author).

Sure, let's leave this in.

In this case, I think we just need to change "Tendry" to "ACR".